mailpoet/views/subscription/admin_user_status_field.php

```diff
@@ -0,0 +1,42 @@
+<?php // phpcs:ignore SlevomatCodingStandard.TypeHints.DeclareStrictTypes.DeclareStrictTypesMissing
```

Comment:
this should be a html file like the others in views and the `MailPoet\Config\Renderer` should be used. You can add variables to the renderer to have some dynamic output

Reply:
Fixed in https://github.com/mailpoet/mailpoet/pull/6147/commits/a0859e62e4e740f00cdb2c69a6e38f0fbaf1ed4c